reporter/src/funTest/assets/evaluated-model-reporter-test-expected-output.json

```diff
@@ -306,14 +311,16 @@
       "path" : "file.java",
       "start_line" : 1,
       "end_line" : 1,
-      "scan_result" : 1
+      "scan_result" : 1,
+      "path_exclude" : 1
```

Comment:
"path_exclude" : 1 should be using a list as there my be multiple path excludes for a single finding

Reply:
I was aware that there maybe multiple matches and I decided to include only one as it is sufficient as a proof, leads to smaller file and is a bit simpler. On the cons side there maybe consistency, but for archiving consistency we could also turn the other list into a single item.
What do you think?

Reply:
I can understand your reasoning, question if this is something ORT users will also expect. I am fine with it for the time being. I presume you pick the first path exclude found if multiple match?

Reply:
I'm also ok changing it, but I wanted to know the reason for doing so.

Reply:
I've turned it into a list.